proto/base_types_pb.js

```diff
@@ -13,7 +13,13 @@
 
```

Comment:
These two files are generated by compiling .proto files.  The .js files seem to be genuinely out of date.